src/lib/Form/Type/URLWildcard/URLWildcardListType.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Form\Type\URLWildcard;
+
+use Ibexa\AdminUi\Form\Data\URLWildcard\URLWildcardListData;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\SearchType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+/**
+ * URLWildcard list form.
+ */
+final class URLWildcardListType extends AbstractType
+{
+    /** @var \Symfony\Contracts\Translation\TranslatorInterface */
+    private $translator;
+
+    public function __construct(TranslatorInterface $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add('type', ChoiceType::class, [
+            'choices' => [
+                $this->translator->trans(/** @Desc("Direct") */ 'url_wildcard.type.direct', [], 'ezplatform_url_wildcard') => true,
+                $this->translator->trans(/** @Desc("Forward") */ 'url_wildcard.type.forward', [], 'ezplatform_url_wildcard') => false,
+            ],
+            'placeholder' => $this->translator->trans(/** @Desc("All") */ 'url_wildcard.type.all', [], 'ezplatform_url_wildcard'),
```

Comment:
Note: Symfony framework does automatically translate these when performing display, both in error display & form blocks (placeholder included).

See `choice_widget_collapsed` in Symfony form templates.

It is only acceptable because of automatic generation of translation resource files, but should be avoided because it entails a second translation attempt during view rendering.

Reply:
@Steveb-p I think it should be ok now, before the upload I checked that everything is displayed correctly and translation_domain points to ezplatform_url_wildcard